.github/scripts/generate_binary_build_matrix.py

```diff
@@ -44,7 +44,7 @@
         "nvidia-curand-cu11==10.3.0.86; platform_system == 'Linux' and platform_machine == 'x86_64' | "
         "nvidia-cusolver-cu11==11.4.1.48; platform_system == 'Linux' and platform_machine == 'x86_64' | "
         "nvidia-cusparse-cu11==11.7.5.86; platform_system == 'Linux' and platform_machine == 'x86_64' | "
-        "nvidia-nccl-cu11==2.21.5; platform_system == 'Linux' and platform_machine == 'x86_64' | "
```

Comment:
Please keep cuda 11.8 builds as is without the change.

Reply:
See https://github.com/pytorch/pytorch/issues/138609

Reply:
You are right. Thanks for catching it. Fixed.